CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) to get started.
 
+## 9.2.0-alpha.1 - April 21, 2020
+### Features
+ - The [`SymbolLayer.iconTextFit`](https://docs.mapbox.com/android/api/map-sdk/9.1.0/com/mapbox/mapboxsdk/style/layers/PropertyFactory.html#iconTextFit-com.mapbox.mapboxsdk.style.expressions.Expression-) property now respects the stretch metadata of any nine-part stretchable image passed into the `style.addImage` method. You can define the stretchable area of an image by configuring the `stretchX`, `stretchY`, `content` options on `Style.addImage` to append the requisite [metadata](https://github.com/mapbox/mapbox-gl-js/issues/8917). ([#314](https://github.com/mapbox/mapbox-gl-native-android/pull/314))
```

Comment:
Should `SymbolLayer.iconTextFit` link to the v9.2.0-alpha.1 docset instead of the v9.1.0 docset, assuming it’ll be published shortly after this changelog goes out?

(We don’t normally link symbols in the iOS and macOS changelogs, because the only place outside GitHub that they appear is in the API reference’s cover page, where jazzy automatically links to the relevant documentation for the current version. I don’t think there’s an analogous place where the Android changelog appears, though.)

Reply:
Yeah, doc for 9.2.0 will be updated after this release.

Reply:
Yeah, great point -- if the 9.2.0 docset will be available after the alpha.1 is published, we should link to that docset instead.